packages/mdc-textfield/foundation.js

```diff
@@ -110,6 +112,7 @@ class MDCTextfieldFoundation extends MDCFoundation {
       this.adapter_.registerTextFieldInteractionHandler(evtType, this.textFieldInteractionHandler_);
     });
     this.adapter_.registerTransitionEndHandler(this.transitionEndHandler_);
+    this.installPropertyChangeHooks_();
```

Comment:
Add these `installPropertyChangeHooks` in the *next* PR. Its a little too hard to review otherwise. And I think that means you don't need the `this.setValueIsProcessing_` field.